client/my-sites/purchases/billing-history/use-redirect-to-history-page-on-wrong-site-for-transaction.tsx

```diff
@@ -16,6 +20,7 @@ export default function useRedirectToHistoryPageOnWrongSiteForTransaction(
 	const doesTransactionMatchSite =
 		doesTransactionExist &&
 		selectedSiteId &&
+		'items' in transaction &&
```

Comment:
This is the other big change in this PR, although if this were not added and we did pass the wrong object type to this component we would have gotten a fatal error, so hopefully it's safe.

Reply:
Looks safe enough to me 👍 Judging by the fact we're already using `transaction.items` as an array, it's fine to have this check to satisfy TS.